chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb

```diff
@@ -63,17 +63,23 @@
 
 elsif node["crowbar_wall"]["crowbar_db_shutdown"]
 
-  # Now, stop the database.
-  # In case of HA setup, also stop remaining pacemaker resources and HA services.
+  # Stop remaining pacemaker resources
+  bash "stop pacemaker resources" do
+    code <<-EOF
+      for type in clone ms primitive; do
```

Comment:
Ah, interesting, I thought we were. I guess it works because we stop one item in the group.